core/src/c/README.txt

```diff
@@ -0,0 +1,5 @@
+The makefile assumes a few things
+
+1) Download and extract the jemalloc artifact into /tmp
```

Comment:
Is it possible to automate this by automatically cloning and building their latest stable release from https://github.com/jemalloc/jemalloc/tree/master ?